ai.py

```diff
@@ -0,0 +1,79 @@
+import math
+import copy
+import random
+
+class AI:
+    def __init__(self):
+        self.symbol = random.randint(1, 2)
+        self.tag = "ai"
+        self.count = 0
+
+    def get_moves(self, board):
+        moves = []
+        for i in range(0, 9):
+            if board.check_move(i):
+                moves.append(i)
+        return moves
+    
+    def make_move(self, board):
+        pass
+
+class MiniMax(AI):
```

Comment:
I would suggest adding some comments to classes. In this particular case the names are self-explanatory, but you shall get used to writing them anyway.